File: python_cgi/admin_panel/delete.php



<?php

include 'connection.php';


$id=$_GET['sid'];

$sql="delete from student where s_id=$id";

$query=mysqli_query($con,$sql) or die("Error in Query");


if($query){
echo "data deleted";
header('Location:dashboard.php');
}else{
echo "Error in deletion";
}

mysqli_close($con);


?>

File: python_cgi/admin_panel/delete.py
#!/usr/bin/python3
import pymysql
import cgi,cgitb
cgitb.enable()
form = cgi.FieldStorage()

print("Content-type:text/html\r\n\r\n")

id=form.getvalue('sid')
db = pymysql.connect(host="localhost",user="root",password="",database="admin_panel" )

cursor = db.cursor()

try:
	cursor.execute("delete from student where s_id=%s" %(id))
	db.commit()
	print("Data Deleted")
except:
    db.rollback()
    print("error in deleting")

db.close()